### ISSUE TITLE
Does it require CGO dependency? Based on the testing process, it seems like it, right?

### ISSUE THREAD

**Issue by wsy998** [BabitMF/bmf-gosdk#1]
Does it require CGO dependency? Based on the testing process, it seems like it, right?

**Comment by sbraveyoung** [comment]
Yes, gosdk is just a set of APIs bound to the go language for golang developers. The c engine layer is also needed for actual operation, so cgo is needed.
